src/Options.tsx

```diff
@@ -12,65 +11,10 @@ export type NavigationBarConfig = {
 
 export type StatusBarConfig = {
     lightStatusBar?: boolean,
-    /**
```

Comment:
That was done in order to show the information to the user, pls check the screenshot bellow:
![Screenshot 2024-05-10 at 14 23 30](https://github.com/inbrainai/sdk-react/assets/59209511/caa724be-0ca0-4b4f-9eb1-1784c9b9bb76)

But with the updated implementation - it's only our internal commnent:
![Screenshot 2024-05-10 at 14 23 41](https://github.com/inbrainai/sdk-react/assets/59209511/774397bf-5297-47e6-b4af-f61c1e88f93a)

Reply:
yes, you are right - fixed now.